packages/synapse-interface/utils/actions/getVirtualPrice.ts

```diff
@@ -0,0 +1,17 @@
+import { readContracts } from '@wagmi/core'
+import SWAP_ABI from '@abis/swap.json'
+
+export const getVirtualPrice = async (poolAddress: string, chainId: number) => {
```

Comment:
this is likely an unnecessary file and can be moved into getPoolData.ts (while still exported), since we handle all other poolData calls there

Reply:
will move

Reply:
moved in https://github.com/synapsecns/sanguine/pull/1023/commits/8b03d2f90d41eee4942e9de3a391a16073f8f230